crates/astria-core-address/src/lib.rs

```diff
@@ -302,9 +302,9 @@ enum ErrorKind {
     },
 }
 
-#[derive(Clone, Copy, Debug)]
+#[derive(Clone, Copy, Debug, Hash)]
```

Comment:
Hah, so I did derive `Hash for Address<T>`, but I didn't derive it for `Bech32` (and `Bech32m`), so you could in fact not use it for `Address` at all.